data.sql

```diff
@@ -9,4 +9,16 @@ values (3,'Pikachu','2021-01-07',15.04,false,1);
 INSERT INTO ANIMALS (id,name,date_of_birth,weight_kg,neutered,escape_attempts)
 values (4,'Devimon','2017-05-12',11.0,true,5);
 
+INSERT INTO ANIMALS (id,name,date_of_birth,weight_kg,neutered,escape_attempts)
+values (5,'Charmander','2020-02-08',11.0,true,0),(6,'Plantmon','2021-11-15',5.7,true,2),
```

Comment:
- [ ] The weight here should be `-11` and not `11` as provided to you in the project requirements.
- [ ] According to the data provided, the neutered field should be marked as false for Charmander.
- [ ] Plantmon should have a weight of `-5.7` and not `5.7`

Reply:
Dropped the table and loaded fresh data accordingly.